### ISSUE TITLE
Select certain folders to sync

### ISSUE THREAD

**Issue by feliciakrismanta** [liberodark/ODrive#30]
I don't know if I missed the setting or not, but it would be great if we are able to sync certain folders only

**Comment by liberodark** [comment]
Hi, Yes is a great idea !

**Comment by NangiDev** [comment]
How is this coming along?

**Comment by liberodark** [comment]
im look that and all other but if you have time to make PR would be great for help my would be great.

**Comment by AMDphreak** [comment]
Yes I agree on this feature. This is a very critical feature. The purpose of Drive is to sync files across many devices, some of which cannot store all of the Drive files on the local disk.
